R/findMotifsInPromoterSeqs.R

```diff
@@ -205,21 +216,21 @@ findMotifsInPromoterSeqs <-
       
       res <- merge(m2, temp, by = c("feature", "seqnames"))
       colnames(res)[6:14] <- paste(name.motif1, colnames(res)[6:14])
-      res <- res[, -c(1, 21, 28)]
+      res <- res[, -c(1, 20)]
```

Comment:
These numbers reflect the resulting column names of `summarizePatternInPeaks `, which is originally written by Julie. Since Junhui and I modified `summarizePatternInPeaks`, I have manually matched those number so that the `findMotifsInPromoterSeqs` function works as expected. I agree that these numbers should be explicitly defined, so, I have added in-context comments describing what the colnames should be.